crates/core/tedge_api/src/commands.rs

```diff
@@ -142,6 +157,12 @@ where
         }
     }
 
+    pub fn payload_from(bytes: &'a [u8]) -> Result<Payload, serde_json::Error> {
+        let mut payload = Payload::from_slice(bytes)?;
+        payload.sanitize();
```

Comment:
I don't get why calling this `sanitize` method so important?

This is about two fields (status and reason) in operation-related messages received at low frequency.

Reply:
If we don't sanitize the payload (esp, removing that duplicate `status` field), then when that payload gets serialized and persisted into the state repository, it'll have two `status` fields. When the same is deserialized on the next startup, it'll cause the parsing to fail.